From 4186100435768f20047aa000fd0e186e7ffed839 Mon Sep 17 00:00:00 2001
From: Sergio Pena <sergio.pena@cloudera.com>
Date: Tue, 22 Nov 2016 14:30:10 -0600
Subject: [PATCH 0833/1164] HIVE-15266: Edit test output of negative blobstore
 tests to match HIVE-15226 (Thomas Poepping,
 reviewed by Sergio Pena and Sahil Takiar)

---
 .../clientnegative/select_dropped_table.q.out      |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/itests/hive-blobstore/src/test/results/clientnegative/select_dropped_table.q.out b/itests/hive-blobstore/src/test/results/clientnegative/select_dropped_table.q.out
index e227b36..ce40c54 100644
--- a/itests/hive-blobstore/src/test/results/clientnegative/select_dropped_table.q.out
+++ b/itests/hive-blobstore/src/test/results/clientnegative/select_dropped_table.q.out
@@ -1,13 +1,13 @@
 PREHOOK: query: CREATE TABLE qtest (key STRING, value STRING)
 #### A masked pattern was here ####
 PREHOOK: type: CREATETABLE
-PREHOOK: Input: #### A masked pattern was here ####
+PREHOOK: Input: ### test.blobstore.path ###/qtest
 PREHOOK: Output: database:default
 PREHOOK: Output: default@qtest
 POSTHOOK: query: CREATE TABLE qtest (key STRING, value STRING)
 #### A masked pattern was here ####
 POSTHOOK: type: CREATETABLE
-POSTHOOK: Input: #### A masked pattern was here ####
+POSTHOOK: Input: ### test.blobstore.path ###/qtest
 POSTHOOK: Output: database:default
 POSTHOOK: Output: default@qtest
 PREHOOK: query: DROP TABLE qtest
-- 
1.7.9.5

